inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -268,6 +271,7 @@ private List<String> getAuditIds(String groupId, String streamId, String sinkNod
         if (sinkNodeType == null) {
             auditSet.add(getAuditId(ClusterType.DATAPROXY, true));
         } else {
+            auditSet.add(getAuditId(sinkNodeType, true));
```

Comment:
Audit supports not only send or receive, please make this interface more common

Reply:
done.